aleph/views/entities_api.py

```diff
@@ -305,10 +305,12 @@ def view(entity_id):
     entity = get_index_entity(entity_id, request.authz.READ, excludes=excludes)
     tag_request(collection_id=entity.get("collection_id"))
     proxy = model.get_proxy(entity)
-    html = proxy.first("bodyHtml", quiet=True)
+    html = proxy.get("bodyHtml", quiet=True)
     source_url = proxy.first("sourceUrl", quiet=True)
     encoding = proxy.first("encoding", quiet=True)
-    entity["safeHtml"] = sanitize_html(html, source_url, encoding=encoding)
+    entity["safeHtml"] = [
+        sanitize_html(value, source_url, encoding=encoding) for value in html
```

Comment:
This is a utility function that we wrote, or a third party created?

Reply:
This is a utility we wrote wrapping the Cleaner class from lxml:

* https://github.com/alephdata/aleph/blob/main/aleph/logic/html.py
* https://lxml.de/api/lxml.html.clean.Cleaner-class.html